Samples/Samples/View/CalendarPage.xaml

```diff
@@ -1,49 +1,172 @@
+    <ResourceDictionary>
+      <converters:CheckBoxEventArgsConverter x:Key="CheckBoxEventArgsConverter"/>
+      <converters:DateSelectedEventArgsConverter x:Key="DateSelectedEventArgsConverter"/>
+      <converters:TimeSelectedEventArgsConverter x:Key="TimeSelectedEventArgsConverter"/>
+      <converters:EpochToDateTimeConverter x:Key="EpochToDateTimeConverter"/>
+      <DataTemplate x:DataType="essentials:IEvent"
+                    x:Key="EventItemTemplate">
+        <ViewCell>
+          <StackLayout>
+            <Grid>
+              <Grid.ColumnDefinitions>
                 <ColumnDefinition/>
                 <ColumnDefinition/>
+                <ColumnDefinition Width="Auto"/>
                 <ColumnDefinition/>
-            </Grid.ColumnDefinitions>
-            <Button Text="Request Read Access"
-                    Command="{Binding RequestCalendarReadAccess}"
-                    HeightRequest="100"/>
-            <Button Grid.Row="0" Grid.Column="1" 
-                    Text="Request Write Access"
-                    Command="{Binding RequestCalendarWriteAccess}"/>
-            <Button Grid.Row="0" Grid.Column="2"  
-                    Text="Load Calendars" 
-                    Command="{Binding GetCalendars}"/>
-        </Grid>
-        <ListView Grid.Row="1" 
-                  ItemsSource="{Binding Calendars}"
-                  ItemTemplate="{StaticResource CalendarItemTemplate}"/>
+              </Grid.ColumnDefinitions>
+              <Label Text="{Binding Title}"
+                     TextColor="Black"
+                     HorizontalTextAlignment="Center"/>
+              <Label Grid.Column="1"
+                     Text="{Binding Start, Converter={StaticResource EpochToDateTimeConverter}}"
+                     TextColor="Black"
+                     HorizontalTextAlignment="Center"/>
+              <Label Grid.Column="2"
+                     Text="-"/>
+              <Label Grid.Column="3"
+                     Text="{Binding End, Converter={StaticResource EpochToDateTimeConverter}}"
+                     TextColor="Black"
+                     HorizontalTextAlignment="Center"/>
+            </Grid>
+          </StackLayout>
+        </ViewCell>
+      </DataTemplate>
+    </ResourceDictionary>
+  </views:BasePage.Resources>
+  <views:BasePage.BindingContext>
+    <viewmodels:CalendarViewModel />
+  </views:BasePage.BindingContext>
+  <views:BasePage.Content>
+    <Grid>
+      <Grid.RowDefinitions>
+        <RowDefinition Height="Auto"/>
+        <RowDefinition Height="Auto"/>
+        <RowDefinition Height="Auto"/>
+        <RowDefinition Height="Auto"/>
+        <RowDefinition/>
+      </Grid.RowDefinitions>
+      <Grid>
+        <Grid.ColumnDefinitions>
+          <ColumnDefinition/>
+          <ColumnDefinition/>
+          <ColumnDefinition/>
+        </Grid.ColumnDefinitions>
+        <Button Text="Request Read Access"
+                Command="{Binding RequestCalendarReadAccess}"
```

Comment:
Just making sure that we're following the naming convention here - should we add a `Command` suffix to the name of it?